README.md

```diff
@@ -117,3 +119,70 @@ It's the main interface of the application. It's responsible for providing the W
     }
   ]
   ```
+
+## For Developers
+
+### Settings environment
+
+To configure your development environment, it's necessary that you have **Pipenv** installed on your machine.
+
+1. First, clone the remote application repository on your local machine:
+
+ ```bash
+ $ git clone https://github.com/rarygc/weather-buddy-api.git
+ ```
+
+2. Them, access the application's base directory:
+
+ ```bash
+ $ cd weather-buddy-api
+ ```
+
+3. Now, create a `.env` environment configuration file in the base directory with the following content:
+
+ ```env
+ # Flask App settings
+ FLASK_APP=api
+ FLASK_ENV=development
+ DEBUG=True
+
+ # OpenWeatherMap API settings
+ OPENWEATHERMAP_TOKEN=3f62164f8ce1f8cb5ae8e2d02918babb
```

Comment:
Usually, exposing a token publicly like this would be bad but, since this token refers to a public tier account, I believe it's fine. Still, it would be advisable for this not the token used in production, thus removing the possibility for someone with access to it (anyone, as this is a public repo) to affect production behavior.

Reply:
Yes, I understand and agree.
An alternative that can be useful for the production environment is the use of the [Secrets](https://docs.github.com/en/actions/reference/encrypted-secrets) tool from GitHub itself. The tests that run on GitHub Actions, for example, are using it to hide my private token.
If a Continuous Delivery script is implemented in the future, it would be a good idea to use this tool also for the production environment.

As the absence of an token directly affects the tests, this exposure of a public token was necessary.
However, guidance for the user to create an account on OpenWeatherMap and provide their own token can be added. This result at more work for run tests and development, but less vulnerable.